demonstrations_v2/tutorial_active_volume/demo.py

```diff
@@ -0,0 +1,656 @@
+To do this without re-introducing idle volume, we need to arrange the logical networks on the
+active volume computer such that they fill up the computational qubits at each time step.
+This can be done by procedurally parallelizing networks via state teleportation, and by splitting
+networks into multiple pieces that are executed sequentially to make them fit into the available
+computational space. We describe teleportation in more detail below but will not dwell on the
+splitting procedure here.
+
+Note that because we had to add vertices to orient the ZX diagram, we require eight additional
+qubits in order to execute the fully parallelized logical network. However, we are not forced
+to maximize the parallelization; as we mentioned above, we instead could have concatenated the
+oriented ZX diagrams for the three individual CNOT gates, or parallelized only two of them, if
+we had less computational space available. This flexibility allows active volume compilation
+to adjust the logical network to the available hardware. In this sense, logical networks form
+a much more flexible representation of operations on an error-correct quantum computer,
+promoting space-time tradeoffs to first-class program transformations.
+
+Under the hood of parallelization: state teleportation
+------------------------------------------------------
+
+In the CNOT ladder example circuit, which has a fundamentally sequential appearance in the
+circuit picture, logical network compilation achieved its parallelization by combining the
+effects of the three CNOTs into one. If we want to parallelize multiple blocks,
+without having to re-compile their joint logical network, we can do this procedurally through
+state teleportation, using a so-called bridge qubit. This technique parallelizes non-commuting
+operations without breaking physics, and it pays off because Bell state preparation and
+measurements are assumed to be fast on an active volume computer.
+
+Physical soundness
+~~~~~~~~~~~~~~~~~~
+
+For the first point, let's consider a somewhat more abstract setup, as is done in Fig. 10b),c)
+in [#Litinski2022]_. Suppose we want to sequentially execute two two-qubit gates :math:`A` and
+:math:`B` that overlap on one qubit and do not commute:
+
+.. figure:: _static/demonstration_assets/active_volume/teleportation_start.png
+    :align: center
+    :width: 40%
+    :target: javascript:void(0)
+
+We also assume that we can track the impact that :math:`B` has on Pauli operators, i.e., what
+happens to a Pauli operator when pulling it through :math:`B`. For our use case, this will be
+easily satisfied, because :math:`B` will be a Clifford operation that simply turns Paulis into
+new Pauli operators.
+
+For the parallelization, we first insert a state teleportation circuit between :math:`A` and
+:math:`B` using a pair of auxiliary qubits prepared in an entangled Bell state, see the
+:doc:`demo on state teleportation <tutorial_teleportation>` for details:
```

Comment:
A bit unfortunate we already have B as an operation label 😅 
```suggestion
:math:`B` using a pair of auxiliary qubits prepared in an entangled Bell state
:math:`|\mathfrak{B}>`, see the :doc:`demo on state teleportation <tutorial_teleportation>`
for details:
```